app/helpers/handlers.js

```diff
@@ -391,12 +390,14 @@ const showPostPage = (currentQuestion, request, h) => {
         setYarValue(request, 'calculatedGrant', 500000)
         // reset SolarPVCost value
         setYarValue(request, 'SolarPVCost', null)
+        setYarValue(request, 'remainingCost', getYarValue(request, 'remainingCostCalf'))
+        setYarValue(request, 'projectCost', getYarValue(request, 'calfHousingCost'))
         return h.redirect('/upgrading-calf-housing/potential-amount-conditional')
         // solar capping
       } else if (getYarValue(request, 'calculatedGrant') > 500000) {
         const solarCap = 500000 - getYarValue(request, 'calculatedGrantCalf')
         // store capped solar value for potential amount solar capped page
-        setYarValue(request, 'calculatedGrantSolarPreCap', getYarValue(request, 'calculatedGrantsolar'))
+        setYarValue(request, 'calculatedGrantSolarPreCap', getYarValue(request, 'SolarPVCost') * 0.25)
```

Comment:
Again, this feels pointless. We have the value of getYarValue(request, 'SolarPVCost') * 0.25 already, which is getYarValue(request, 'calculatedGrantSolar')